src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://localhost:8080/api"
+      data-qa="big"
+    >
+      <label class="search search__large" for="searchInput">
```

Comment:
why do you need this label?

Reply:
> why do you need this label?

so we can click everywhere and get into the intut, is that wrong?